cloud/deployment/fromfile/fromfile_test.go

```diff
@@ -126,6 +126,7 @@ deployment:
     scheduler_au: 5
     scheduler_count: 3
     cluster_id: cluster-id
+    deployment_type: HYBRID
```

Comment:
just a curious question, what's the official customer-facing name of the product, is it HYBRID or BRING_YOUR_OWN_CLOUD? I have seen reference for both in our codebase

Reply:
Customer Facing Name is HYBRID. BRING_YOUR_OWN_CLOUD, is mostly a type for clusters and HYBRID as type for deployments